test/python/primitives/test_estimator.py

```diff
@@ -58,358 +58,6 @@ def setUp(self):
             [1, 2, 3, 4, 5, 6],
         )
 
-    def test_estimator(self):
```

Comment:
Just to double check, is there anything we're removing here that should be ported to use whatever the proper new code paths are? In other words, are we inadvertently reducing code coverage?

Reply:
Thank you. I checked all tests. I readded three tests: `test_run_with_operator` and `test_run_with_shots_option_none` for Estimator and `test_run_reverse_meas_order` for Sampler.

1. I removed duplicated tests.
2. We don't test e2e test for PauliSumOp, instead we added the conversion test (`test_validate_observables`). 
3. No tests corresponding to `test_estimator_param_reverse` since we do not have this feature (Passing parameters, not `parameter_values`).